test/e2e/utils/exec.go

```diff
@@ -29,7 +29,7 @@ type ExecOptions struct {
 // returning stdout, stderr and error. `options` allowed for
 // additional parameters to be passed.
 func ExecWithOptions(client corev1client.CoreV1Interface, options ExecOptions) (string, string, error) {
-	config := framework.TestContext.RestConfig
+	config := framework.TestContext.RestConfigs[0]
```

Comment:
Why the first one? This limits executing on pods to first kube cluster. We should extract it to parameter.

Reply:
I wanted to avoid making any changes to existing tests, but ok, I guess we can pass a framework through parameters